packages/frontend/src/pages/bridge-details/BridgeDetailsPage.tsx

```diff
@@ -28,7 +31,7 @@ export const BridgeDetailsPage = () => {
     sleepSeconds: 9,
     startImmediate: true,
     callback: useCallback(() => setSeed(Date.now()), [setSeed]),
-    onTick: setTimer,
+    onTick: useCallback(setTimer, []),
```

Comment:
`setTimer` is a `useState`-setter. It should not be necessary to wrap it with `useCallback`.